r_code/jackknife_te.R

```diff
@@ -34,12 +41,21 @@ pacman::p_load(dplyr, furrr, stringr, tidyr, tibble, resample)
           !!as.name(counterfac_var))
       ),
       !!as.name(cf_pct_col) := dplyr::case_when(
-        effect_type == "aggregate" ~ (sum(!!as.name(outcome_var)) /
-          sum(!!as.name(counterfac_var))) - 1,
-        effect_type == "mean" ~ mean((!!as.name(outcome_var) /
-          !!as.name(counterfac_var)) - 1),
-        effect_type == "median" ~ median((!!as.name(outcome_var) /
-          !!as.name(counterfac_var)) - 1)
+        effect_type == "aggregate" ~ 
+          ifelse(sum(!!as.name(counterfac_var))==0, NA_real_,
+                 (sum(!!as.name(outcome_var)) /
+                    sum(!!as.name(counterfac_var))) - 1  
```

Comment:
It might be useful to also have a helper function for this common case, too.

Reply:
I could recreate the CarefulDivision and use it throughout other scripts.